develop/dev-guide-bi-looker-studio.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Connect to TiDB Cloud with Looker Studio
+summary: Learn how to connect to TiDB Cloud using Looker Studio
+---
+
+# Connect to TiDB Cloud with Looker Studio
+
+TiDB is a MySQL-compatible database, and [Looker Studio](https://lookerstudio.google.com/) is a free web-based BI tool that can visualize data from various sources. 
+
+In this tutorial, you can learn how to connect to your TiDB cluster using Looker Studio.
+
+> **Note:**
+>
+> - For this tutorial we will use a serverless cluster. Looker Studio can also be used with dedicated clusters.
+
+## Prerequisites
+
+To complete this tutorial, you need:
+
+- Google Account. 
+- A TiDB cluster.
+
+**If you don't have a TiDB cluster, you can create one as follows:**
+
+- Follow [Creating a TiDB Serverless cluster](/develop/dev-guide-build-cluster-in-cloud.md) to create your own TiDB Serverless cluster.
+
+## Import Dataset
+
+You can use the S&P 500 dataset provided as an Interactive Tutorial of TiDB Serverless. 
+
+<SimpleTab>
+<div label="TiDB Serverless">
+
+1. Navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page, and click the question mark icon in the bottom-right corner. A Help dialog is displayed. 
+
+2. Click **Interactive Tutorials**. Then click **S&P 500 Analysis**.
+
+3. In the **Import S&P 500 Dataset** dialog, select your Serverless cluster. Then click the **Import Dataset** button. It will start importing briefly.
+
+4. After the status changed to **Imported**, you can close this dialog. Click the **Exit Tutorial** button in the bottom-left corner of the dialog. 
+
+</div>
+</SimpleTab>
+
+## Connect to TiDB cluster
+
+<SimpleTab>
+<div label="TiDB Serverless">
+
+1. Navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page, and then click the name of your target cluster to go to its overview page.
+
+2. Click **Connect** in the upper-right corner. A connection dialog is displayed.
+
+3. Set **Operating System** to `Windows` in the connection dialog. Then Set **Connect With** to `General`. You can download [CA cert](https://letsencrypt.org/certs/isrgrootx1.pem) at the note below. 
```

Comment:
@qiancai Unfortunately not. Since Looker Studio is a SaaS, we need uploadable ca file to enable SSL connection on Looker Studio. Explicitly setting Operating System as Windows, we can download ca.file.